source/state_representation/include/state_representation/space/cartesian/CartesianAcceleration.hpp

```diff
@@ -272,13 +282,27 @@ class CartesianAcceleration : public CartesianState {
    */
   CartesianAcceleration& operator+=(const CartesianAcceleration& acceleration);
 
+  /**
+   * @brief Overload the += operator with a state
+   * @param state The Cartesian state to add to
+   * @return The current Cartesian acceleration added the Cartesian state given in argument
```

Comment:
I know you are following the existing style but these docstrings could be improved. Since it is being replicated in many places I think it's in scope of the PR to touch up a bit. Because addressing `-`, `*` and `/` operators is also planned in future PRs, I think it would be good to amend the relevant docstrings as part of those refactors. Otherwise, if you disagree, rewriting the docs can be a standalone PR. However I feel that we will probably just forget to do it in that case.

My issues line by line: 
- the brief is not helpful because it just restates the functional syntax. It should instead describe the behavior of the operation
- in the parameter description, the preposition is misleading. The input argument is being added to the current instance, not the other way around (`a += b` is the same as `a = a + b`) which matters for orientational terms.
- the wording of the return statement is also very confusing

I would suggest something along these lines for increased clarity and precision, though I'm open to other wordings of course.
```suggestion
   * @brief Add the acceleration of another state
   * @param state A Cartesian state in the same reference frame
   * @return The combined Cartesian acceleration
```